ddtrace/contrib/redis/patch.py

```diff
@@ -119,6 +121,20 @@ def unpatch():
                 unwrap(redis.asyncio.cluster.ClusterPipeline, "execute")
 
 
+def _run_redis_command(span, func, args, kwargs):
```

Comment:
General question, is this how we want to standardize all integrations moving forward (tracing in `ddtrace/_trace/...`, patch method in `patch.py`, and any helper functions in `utils.py`?

Reply:
Yes, so far it's my best guess at an ideal separation of concerns.